### PR TITLE
[admin] More admin only ghost spawners!

### PR DESCRIPTION
**Pull request by Bibby0110**
### Intent of your Pull Request
MORE admin only ghost spawners for minievents/bussing/whatever

(I am bad at GitHub, shout at me)

#### Changelog

:cl:  
rscadd: Added new admin spawner things  
/:cl:

### CODE REVIEW

File: yogstation/code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -101,4 +101,160 @@
...
\ No newline at end of file
+		return null
+		
+		
+		
+		
+		
+		
+		
+
```
Comment (monster860) [inline yogstation/code/game/objects/structures/ghost_role_spawners.dm:105]: aaAaaAAAA thats a lot of empty lines

**Review by monster860 — CHANGES_REQUESTED** [review]

File: yogstation/code/game/objects/structures/ghost_role_spawners.dm (hunk 1 above)
Comment (Bibby0110) [reply yogstation/code/game/objects/structures/ghost_role_spawners.dm:105]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

File: yogstation/code/game/objects/structures/ghost_role_spawners.dm [hunk 2]
```diff
@@ -101,4 +101,153 @@
...
+	name = "grown alien egg"
+	desc = "A pulsating alien egg. You can barely recognise what looks to be an alien larva inside."
+	mob_name = "an alien larva"
+	icon = 'icons/mob/alien.dmi'
+	icon_state = "egg"
+	roundstart = FALSE
+	death = FALSE
+	random = TRUE
+	mob_type = /mob/living/carbon/alien/larva
+	flavour_text = "<span class='big bold'>You are an alien.</span><b> You awoke on this metal bird. You have been in statis \
```
Comment (Amelia0010) [inline yogstation/code/game/objects/structures/ghost_role_spawners.dm]: statis 🤔

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

File: yogstation/code/game/objects/structures/ghost_role_spawners.dm (hunk 2 above)
Comment (Bibby0110) [reply yogstation/code/game/objects/structures/ghost_role_spawners.dm]: Web edit: Completed

**Review by Altoids1 — APPROVED** [review]

**Pull request merged**